### ISSUE TITLE
global problem which my explain the symptoms (last issues posted by me)

### ISSUE THREAD

**Issue by evanescente-ondine** [uzbl/uzbl#396]
Errors of uzbl-browser:
https://pastebin.com/DJVQ77j6
and of uzbl: https://pastebin.com/
Looooot of info you might appreciate. I did compile it. Many bindings don't work, but I am sure it's only the surface.

=> "uzbl-core" \--bug-info
Commit: v0.9.1-191-g676b47e3
GTK compile: 3.22.11
GTK run: 3.22.11
WebKit compile: 2.16.6
WebKit run: 2.16.6
WebKit2: 1
libsoup compile: 2.56.0
libsoup run: 2.56.0

**Comment by keis** [comment]
yeah, that looks bad. Is that from starting uzbl or are you doing something more?

Please try to get a stacktrace when running with
`G_DEBUG=fatal-criticals uzbl-core`

**Comment by evanescente-ondine** [comment]
ok, but how I am supposed to run this ? "uzbl-core" D_DEBUG=fatal-criticals (the quote marks are for my shell) ? It doesn't seem to work like that.
Either way: => "uzbl-core" G_DEBUG=fatal-criticals -v
error on run_system_command: L'exécution du processus fils « /home/mehdi/.local/share/uzbl:/uzbl:/share/uzbl/examples/data:scripts/load_cookies.sh » a échoué (Aucun fichier ou dossier de ce type)
error on run_system_command: L'exécution du processus fils « /home/mehdi/.local/share/uzbl:/uzbl:/share/uzbl/examples/data:scripts/load_cookies.sh » a échoué (Aucun fichier ou dossier de ce type)
Uzbl start location: /usr/local/bin/uzbl-core
window_id 37748739
pid 2000
name: 2000
commit: v0.9.1-191-g676b47e3
error on run_system_command: L'exécution du processus fils « /home/mehdi/.local/share/uzbl:/uzbl:/share/uzbl/examples/data:scripts/scheme.py » a échoué (Aucun fichier ou dossier de ce type)
error on run_system_command: L'exécution du processus fils « /home/mehdi/.local/share/uzbl:/uzbl:/share/uzbl/examples/data:scripts/scheme.py » a échoué (Aucun fichier ou dossier de ce type)
